exercises/e001/results/Frydan/b32gen.py

```diff
@@ -0,0 +1,32 @@
+#!/bin/python3
+
+import argparse
+import random
+
+parser = argparse.ArgumentParser(description='Base32 (RFC 4648) random character string generator')
+meg = parser.add_mutually_exclusive_group()
+parser.add_argument('characters', type=int, default=32, nargs='?', help='Number of characters to generate (default: 32)')
```

Comment:
```suggestion
parser.add_argument('characters', type=int, default=32, nargs='?', help='Number of characters to generate (default: 32)')
meg = parser.add_mutually_exclusive_group()
```

This is nitpicking, but I would swap lines around.

Reply:
Just to visually separate mutually exclusive group from other arguments or is there another reason for it?

Reply:
This is a bit of bike-shedding, so don't worry or think too much about it IMO. I would swap it around as I would out line after line, objects which I'm modifying together. Currently you have:

```
parser = argparse.A...
meg = parser.add_mu...
parser.add_argume...
meg.add_argume...
meg.add_ar...
```

I would personally wrote:
```
parser = argparse.A...
meg = parser.add_mu...
meg.add_argume...
meg.add_ar...
```